README.md

```diff
@@ -3,9 +3,24 @@
 Tool for checking status of alembic migrations inside [Apache
```

Comment:
@andriisoldatenko this tagline could be updated to be more general now that we have multiple scripts.

Reply:
nice catch on it.

Reply:
fixed in a661eb5